libmaxminddb/plan.sh

```diff
@@ -0,0 +1,72 @@
+pkg_origin=core
+pkg_name=libmaxminddb
+pkg_version=1.3.2
+pkg_license=('Apache')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="https://github.com/maxmind/libmaxminddb/releases/download/${pkg_version}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum="e6f881aa6bd8cfa154a44d965450620df1f714c6dc9dd9971ad98f6e04f6c0f0"
+pkg_upstream_url="https://github.com/maxmind/libmaxminddb"
+pkg_description="C library for the MaxMind DB file format"
+pkg_build_deps=(
+  core/coreutils
+  core/gcc
+  core/make
+  core/perl
+  )
+pkg_deps=(core/glibc)
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+pkg_include_dirs=(include)
+
+geolite_city_filename="GeoLite2-City.tar.gz"
+geolite_city_archive="https://geolite.maxmind.com/download/geoip/database/${geolite_city_filename}"
+geolite_city_sha256sum="589b8603a6cd98cf134f96de24618ba1d986a954e40409a67f398eb1edbf6084"
+
+geolite_country_filename="GeoLite2-Country.tar.gz"
+geolite_country_archive="https://geolite.maxmind.com/download/geoip/database/${geolite_country_filename}"
+geolite_country_sha256sum="1b627bd7a575500cbf9e675630ab2c9b1632c3727f3eab29a06e50568341bdfa"
+
+geolite_asn_filename="GeoLite2-ASN.tar.gz"
+geolite_asn_archive="https://geolite.maxmind.com/download/geoip/database/${geolite_asn_filename}"
+geolite_asn_sha256sum="6fc0855cdc3514c5b7cf19fe1a2681c5b1bbc5009c3a89b1453b64c8c235ba2a"
+
+do_check() {
+  make check
+}
+
+do_prepare() {
+  interpreter_old="/usr/bin/env"
+  interpreter_new="$(pkg_path_for coreutils)/bin/env"
+
+  grep -nrlI '\#\!/usr/bin/env' "$HAB_CACHE_SRC_PATH/${pkg_dirname}/t" | while read -r filesToFix; do
+    sed -e "s#\#\!${interpreter_old}#\#\!${interpreter_new}#" -i "$filesToFix"
+  done
+
+  ./configure --prefix="$pkg_prefix"
+}
+
+do_install() {
+  do_default_install
+
+  local geoip_install_path="${pkg_prefix}/share/GeoIP"
+
+  mkdir -p "${geoip_install_path}"
+  cd "${geoip_install_path}"
+
+  build_line "Installing GeoIP City Files"
+  download_file $geolite_city_archive "${geoip_install_path}/${geolite_city_filename}" $geolite_city_sha256sum
```

Comment:
I've got mixed feelings on this.  One one hand, this software requires a geodatabase in order to be functional so adding these files makes sense. However, it expects to always be passed `--file /path/to/database`, so we could track each geodatabase as a separate package and leave it up to the user to ensure that libmaxmind and the database are in their pkg_deps.

I'm not sure we want to be tracking data files in core plans though, though I know there is one plan (goaccess?) that needs them.  

I have a few suggestions depending on the direction we go,  but I'll hold off on them until we make that decision.

Reply:
I understand the want for modularity, but it just seems to be pointless in this instance, since we can't use the binary without the database files. If we're going to have other plans that will need them, like goaccess, it would make sense to make two separate plans. Just let me know what the consensus is and I'll get it done :)

Reply:
I think in this case, the alternative of tracking "data only" packages feels like a worse solution than bundling a minimal set of data in this package, so I'm good with the approach in this plan.  I'm going to mark this conversation as resolved and leave additional changes as a separate item.